packages/patternfly-3/patternfly-react/sass/patternfly-react/_date-time-picker.scss

```diff
@@ -0,0 +1,74 @@
+.bootstrap-datetimepicker-widget.date-time-picker-pf.top {
+  margin-top: -10px;
+}
+
+.bootstrap-datetimepicker-widget.date-time-picker-pf.bottom {
+  margin-top: 10px !important;
+}
+
+.bootstrap-datetimepicker-widget.date-picker-pf,
+.bootstrap-datetimepicker-widget.date-time-picker-pf {
+  max-width: none;
+
+  .arrow {
+    left: 10% !important;
```

Comment:
please try not to use `!important` boith here and also in other places, 
probably we'll need to modify patternfly-core css to make this component more adjustable.

Reply:
The arrow position is set in react-bootstrap's component and can't be changed in a different way

Reply:
maybe you can open an issue or PR to `react-bootstrap` and link it here?

Reply:
its fixed in the newer versions of react-bootstrap